pkg/utils/annotations.go

```diff
@@ -15,18 +15,38 @@ limitations under the License.
 
 package utils
 
-import "github.com/fluid-cloudnative/fluid/pkg/common"
+import (
+	"github.com/fluid-cloudnative/fluid/pkg/common"
+	stdlog "log"
+	"os"
+)
+
+var (
+	ServerlessPlatformKey string = ""
+	ServerlessPlatformVal string = ""
+)
+
+func init() {
+	if envVal, exists := os.LookupEnv(common.EnvServerlessPlatformLabel); exists {
+		ServerlessPlatformKey = envVal
+		stdlog.Printf("Found %s value %s, using it as ServerlessPlatformLabelKey", common.EnvServerlessPlatformLabel, envVal)
+	}
+	if envVal, exists := os.LookupEnv(common.EnvServerlessPlatformVal); exists {
+		ServerlessPlatformVal = envVal
+		stdlog.Printf("Found %s value %s, using it as ServerlessPlatformLabelValue", common.EnvServerlessPlatformVal, envVal)
+	}
+}
 
 func ServerlessEnabled(infos map[string]string) (match bool) {
-	return matchedValue(infos, common.ServerlessPlatform, common.ASKPlatform) || enabled(infos, common.InjectServerless) || enabled(infos, common.InjectFuseSidecar)
+	return matchedValue(infos, ServerlessPlatformKey, ServerlessPlatformVal) || enabled(infos, common.InjectServerless) || enabled(infos, common.InjectFuseSidecar)
```

Comment:
What will happen if the ServerlessPlatformKey and ServerlessPlatformVal is not set?

Reply:
Have add an extra check empty string process. If not set, it returns false.